microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -596,16 +596,23 @@ def verify_repository_installed(self, node: Node) -> None:  # noqa: C901
                     "update` output",
                 ).is_true()
             else:
-                is_repository_configured_correctly = any(
-                    [
-                        "deb.debian.org" in repository.uri
-                        or "debian-archive.trafficmanager.net" in repository.uri
-                        for repository in debian_repositories
```

Comment:
add this `or "azure.deb.debian.cloud/debian" in repository.uri' is enough?

Reply:
Comparison of version > 12 is not required?

Reply:
not required, if repo can match any one, test pass